Jenkinsfile_CNP

```diff
@@ -160,4 +155,14 @@ withPipeline(type, product, component) {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'output/**/*'
     publishToCosmosDb(this, params, 'dbs/jenkins/colls/fpl-test-metrics', "${WORKSPACE}/output", 'metrics')
   }
+
+  after('buildinfra:demo') {
+    env.URL = "https://ccd-case-management-web-demo.service.core-compute-demo.internal"
+    importCoreCaseDataDefinitions('demo')
+  }
+
+  after('aat:promotion') {
+    env.URL = "https://ccd-case-management-web-aat.service.core-compute-aat.internal"
+    importCoreCaseDataDefinitions('aat')
```

Comment:
importCoreCaseDataDefinitions inside is doing
eval \$(./bin/variables/load-${env}-environment-variables.sh)
but load-aat-environmnet-variables.sh does not exists

Reply:
adding :)